screencapture.html

```diff
@@ -0,0 +1,40 @@
+<!doctype html>
+<!--
+ Copyright 2024 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/src/style.css" />
+    <script type="module" src="/src/components/scripts/screencapture.ts"></script>
+    <title>IWA Kitchen Sink - Screen capture</title>
+  </head>
+  <body>
+    <div class="container screencapture">
+      <!-- Load shared nav -->
+      <load src="src/components/nav.html" active="Screen capture" />
+      
+      <button id="captureScreenButton" style="width: 150px; height: 50px;">Capture the screen</button>
```

Comment:
Two comments here:
* Please use CSS to style elements, don't write inline style attributes
* Same nit as with `captureScreenDiv`; something as simple as `capture` would work here, although `button` has enough semantic meaning that something like `capture--button` would be appropriate.